faq/manage-cluster-faq.md

```diff
@@ -427,6 +427,6 @@ This section describes common problems you may encounter during backup and resto
 
 ### How to back up data in TiDB?
 
-Currently, for the backup of a large volume of data, the preferred method is using [BR](/br/backup-and-restore-tool.md). Otherwise, the recommended tool is [Dumpling](/dumpling-overview.md). Although the official MySQL tool `mysqldump` is also supported in TiDB to back up and restore data, its performance is worse than [BR](/br/backup-and-restore-tool.md) and it needs much more time to back up and restore large volumes of data.
+Currently, for the backup of a large volume of data, the preferred method is using [BR](/br/backup-and-restore-overview.md). Otherwise, the recommended tool is [Dumpling](/dumpling-overview.md). Although the official MySQL tool `mysqldump` is also supported in TiDB to back up and restore data, its performance is no better than BR and it needs much more time to back up and restore large volumes of data.
```

Comment:
“A large volume of data” is ambiguous. Need a clear description here.

Reply:
Fixed in https://github.com/pingcap/docs/pull/9206/files#diff-99d1372119ae4ed697e8d7fda222067f376d7bc2881ddce7d4bec353c2d00b05R430.